### PR TITLE
[Dem/structures] Python updated to current standard

### PR DESCRIPTION
**Pull request by ipouplana**
Summary:

- Python import warning addressed
- Python files updated to work with new DEM analysis stage
- Cleanup of python main algorithm

### CODE REVIEW

File: applications/DemStructuresCouplingApplication/python_scripts/sp_dem_fem_coupling_algorithm.py
Comment (philbucher) [inline applications/DemStructuresCouplingApplication/python_scripts/sp_dem_fem_coupling_algorithm.py]:
```suggestion
```

**Review by philbucher — CHANGES_REQUESTED** [review]
can you please reduce the size of the files you are adding?
If this is not possible then please move those cases to the examples repo

**Comment by ipouplana** [comment]
Done @philbucher , I just moved the test examples to the Examples repo (PR 52)

**Comment by philbucher** [comment]
thanks!

I will approve once you have approval from someone that has knowledge about this app

**Review by maceligueta — APPROVED** [review]

**Comment by maceligueta** [comment]
Great @ipouplana . Thanks for adapting/cleaning these files

**Review by philbucher — APPROVED** [review]

**Pull request merged**
